admin/console.rst

```diff
@@ -51,6 +51,29 @@ This reduces loading times greatly.
 
 .. include:: /admin/console-rails-shell.include.rst
 
+.. warning:: **Safe Mode**
+
+   Normally, starting rails console requires certain `third-party services`_ to
+   be up and running. You may receive errors and console will refuse to start
+   in case they are not available.
+
+   However, it's possible to start rails console in safe mode by setting a
+   special environment variable. With ``ZAMMAD_SAFE_MODE=1`` set, availability
+   of these services will be ignored:
+
+   .. code-block:: console
+
+      $ ZAMMAD_SAFE_MODE=1 rails c
```

Comment:
This is a very fixed example for source code and docker variants only. Maybe wildcarding it or something might be a good idea. Or at least note that it depends on the used system.

People tend to be *very* confused on what variant to use especially if you don't push their head into the differences.

Reply:
Fixed.

Switched to using tabs with separate code blocks.